src/Components/Patient/FileUpload.tsx

```diff
@@ -1115,183 +1121,184 @@ export const FileUpload = (props: FileUploadProps) => {
-                    fill="currentColor"
-                    d="M4 12a8 8 0 018-8V0C5.373 0 0 5.373 0 12h4zm2 5.291A7.962 7.962 0 014 12H0c0 3.042 1.135 5.824 3 7.938l3-2.647z"
-                  ></path>
-                </svg>
-                Proceed
-              </button>
-              <button
-                type="button"
-                className="btn-danger btn mr-2 w-full md:w-auto"
-                disabled={btnloader}
-                onClick={(_) => setModalOpenForEdit(false)}
-              >
-                Cancel
-              </button>
+            <div className="m-4">
+              <h1 className="text-black text-xl "> Edit File Name</h1>
             </div>
-          </form>
-        </div>
-      </Modal>
-      <Modal open={modalOpenForArchive}>
-        <div className="h-screen w-full absolute flex items-center justify-center ">
-          <form
-            onSubmit={(event: any) => {
-              event.preventDefault();
-              setbtnloader(true);
-              archiveFile(modalDetails?.id, archiveReason);
-            }}
-            className="bg-white rounded shadow p-8 m-4 max-h-full flex flex-col max-w-lg w-2/3 min-w-max-content"
-          >
-            <div className="text-center m-2">
-              <i className="fa-solid fa-5x fa-triangle-exclamation text-red-500"></i>
+          </div>
+        }
+        onClose={() => setModalOpenForEdit(false)}
+      >
+        <form
+          onSubmit={(event: any) => {
+            event.preventDefault();
+            setbtnloader(true);
+            partialupdateFileName(modalDetails?.id, editFileName);
+          }}
+          className="flex flex-col w-full"
+        >
+          <div>
+            <InputLabel
+              className="text-md text-gray-500"
+              id="editfilenamelabel"
+            >
+              Please enter the file name
+            </InputLabel>
+            <TextInputField
+              name="editFileName"
+              variant="outlined"
+              margin="dense"
+              value={editFileName}
+              onChange={(e) => setEditFileName(e.target.value)}
+              errors={editFileNameError}
+            />
+          </div>
+          <div className="flex flex-col-reverse md:flex-row gap-2 mt-4 justify-end">
+            <ButtonV2
+              variant="primary"
+              type="submit"
+              disabled={btnloader}
+              className="mr-2 w-full md:w-auto"
+            >
+              <i
+                className={`fa-solid fa-spinner mr-2 animate-spin ${
+                  !btnloader ? " hidden " : " "
+                }`}
+              ></i>
+              Proceed
+            </ButtonV2>
+            <ButtonV2
+              variant="secondary"
+              type="button"
+              className="mr-2 w-full md:w-auto"
+              disabled={btnloader}
+              onClick={() => setModalOpenForEdit(false)}
+            >
+              Cancel
+            </ButtonV2>
```

Comment:
Merge the latest develop and use the generalized `<Cancel onClick={...} /` button.